api/command_test.go

```diff
@@ -276,8 +290,8 @@ func TestTestCommand(t *testing.T) {
 
 	time.Sleep(100 * time.Millisecond)
 
-	p2 := Client.Must(Client.GetPosts(channel1.Id, 0, 2, "")).Data.(*model.PostList)
-	if len(p2.Order) != 2 {
+	p2 := Client.Must(Client.GetPosts(channel1.Id, 0, 10, "")).Data.(*model.PostList)
```

Comment:
why fetch 10 if you only need 3?

Reply:
Original test checks for exact length value which now at 3. Setting page limit to 3 will narrow checks at upper limit. That's why I've set to 10 so that it will fail just in case it returned 4 or more posts. It might not be necessary today but maybe in the future.